velox/experimental/cudf/exec/CudfHashJoin.cpp

```diff
@@ -22,7 +22,6 @@
 #include "velox/exec/Task.h"
 
 #include <cudf/copying.hpp>
-#include <cudf/join.hpp>
```

Comment:
This header was used for `cudf::hash_join`. Following “include what you use” we should keep the needed headers, even if they are transitively included by `CudfHashJoin.h`. The names may need to change like in the corresponding header.